Source/WebCore/css/parser/CSSTokenizerInputStream.cpp

```diff
@@ -45,11 +47,31 @@ void CSSTokenizerInputStream::advanceUntilNonWhitespace()
     // Using ASCII whitespace here rather than CSS space since we don't do preprocessing
     if (m_string->is8Bit()) {
         auto characters = m_string->span8();
-        while (m_offset < m_stringLength && isASCIIWhitespace(characters[m_offset]))
+        while (m_offset < m_stringLength && isASCIIWhitespace(characters[m_offset])) {
+            if (CSSTokenizer::isNewline(characters[m_offset]))
```

Comment:
I am confused now: Given the name of this function, it should not stop at a newline. If we are changing its behavior we should change its name too.

Reply:
Fixed, yup I was got confused as well with the renaming, thanks for catching it.